staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -34,6 +34,16 @@ type CustomResourceDefinitionSpec struct {
 	Validation *CustomResourceValidation
 	// Subresources describes the subresources for CustomResources
 	Subresources *CustomResourceSubresources
+
+	// Columns
+	Columns []Column
+}
+
+type Column struct {
```

Comment:
CustomResourceColumnDefinition

Reply:
To match `TableColumnDefinition` in meta/v1beta1

Reply:
done